apps/todo/app/page.tsx

```diff
@@ -1,144 +1,52 @@
-  conic,
-  className,
-  small,
-}: {
-  small?: boolean;
-  conic?: boolean;
-  className?: string;
-}): JSX.Element {
-  return (
-    <span
-      className={[
-        styles.gradient,
-        conic ? styles.glowConic : undefined,
-        small ? styles.gradientSmall : styles.gradientLarge,
-        className,
-      ]
-        .filter(Boolean)
-        .join(" ")}
-    />
-  );
-}
+import { FC, useState } from "react";
+import { useForm, SubmitHandler } from "react-hook-form";
 
-const LINKS = [
-  {
-    title: "Docs",
-    href: "https://turbo.build/repo/docs",
-    description: "Find in-depth information about Turborepo features and API.",
-  },
-  {
-    title: "Learn",
-    href: "https://turbo.build/repo/docs/handbook",
-    description: "Learn more about monorepos with our handbook.",
-  },
-  {
-    title: "Templates",
-    href: "https://turbo.build/repo/docs/getting-started/from-example",
-    description: "Choose from over 15 examples and deploy with a single click.",
-  },
-  {
-    title: "Deploy",
-    href: "https://vercel.com/new",
-    description:
-      "Instantly deploy your Turborepo to a shareable URL with Vercel.",
-  },
-];
+type Input = {
+  item: string;
+};
 
-export default function Page(): JSX.Element {
-  return (
-    <main className={styles.main}>
-      <div className={styles.description}>
-        <p>
-          examples/basic&nbsp;
-          <Code className={styles.code}>web</Code>
-        </p>
-        <div>
-          <a
-            href="https://vercel.com?utm_source=create-turbo&utm_medium=basic&utm_campaign=create-turbo"
-            rel="noopener noreferrer"
-            target="_blank"
-          >
-            By{" "}
-            <Image
-              alt="Vercel Logo"
-              className={styles.vercelLogo}
-              height={24}
-              priority
-              src="/vercel.svg"
-              width={100}
-            />
-          </a>
-        </div>
-      </div>
+type Todo = {
+  id: number | null;
+  item: string;
+};
 
-      <Button appName="web" className={styles.button}>
-        Click me!
-      </Button>
+const TodoPage: FC = () => {
+  const [todoList, setTodoList] = useState<Todo[]>([]);
+  const { register, handleSubmit, reset } = useForm<Input>();
 
-      <div className={styles.hero}>
-        <div className={styles.heroContent}>
-          <div className={styles.logos}>
-            <div className={styles.circles}>
-              <Image
-                alt=""
-                height={614}
-                src="circles.svg"
-                width={614}
-                style={{ pointerEvents: "none" }}
-              />
-            </div>
-            <div className={styles.logoGradientContainer}>
-              <Gradient className={styles.logoGradient} conic small />
-            </div>
+  const onSubmit: SubmitHandler<Input> = (data) => {
+    setTodoList(() => [...todoList, { id: todoList.length, item: data.item }]);
+    reset();
+  };
```

Comment:
こうしちゃえば`SubmitHandler`型はimportしなくで済みます
```suggestion
  const onSubmit = handleSubmit((data) => {
    setTodoList(() => [...todoList, { id: todoList.length, item: data.item }]);
    reset();
  });
```

Reply:
修正しました！
[36be738](https://github.com/peeek-develop-training/catchup-front/pull/4/commits/36be738f665914da60544d01e560a2475ecdde2e)